.gitignore

```diff
@@ -18,7 +18,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
-
+.idea

```

Comment:
This is an environment specific hidden file for macOS. This shouldn't be committed. You can ignore this in your global `.gitignore`

Reply:
Ok, I revolved issue, thanks for noting it